action/protocol/poll/protocol.go

```diff
@@ -197,6 +198,24 @@ func (p *governanceChainCommitteeProtocol) Initialize(
 	log.L().Info("Initialize poll protocol", zap.Uint64("height", p.initGravityChainHeight))
 	var ds state.CandidateList
 	if ds, err = p.delegatesByGravityChainHeight(p.initGravityChainHeight); err != nil {
+		errMsg := "bucket = electionNS doesn't exist: not exist in DB"
+		errChan := make(chan error, 1)
+		errChan <- err
```

Comment:
You don't have to check error asynchronously. All could happen in a synchronous loop with sleep backoff.

Reply:
sure